### ISSUE TITLE
ISBN

### ISSUE THREAD

**Issue by ClusterWood** [hans/obsidian-citation-plugin#193]
Thank you for the great plugin. Unfortunately, the ISBN field is not transferred. Is it possible to add this?

**Comment by ClusterWood** [comment]
Thank you for the great plugin. Unfortunately, the ISBN field is not transferred. Is it possible to add this?

**Comment by orionpilot** [comment]
If a biblatex field exists in your entry and does not have a shortcode with `{{field}}` for example, than you may access it with `{{entry.data.fields.FIELD}}`. In the ISBN case, you can access it with `{{entry.data.fields.isbn}}` in the template settings of the plugin.

**Comment by ClusterWood** [comment]
Yes, this works in principle, but not if you use 'Citation' in conjunction with 'QuickAdd' and 'Templater', e.g. as described here:
https://bagerbach.com/blog/how-i-read-research-papers-with-obsidian-and-zotero

**Comment by orionpilot** [comment]
Ah, that's interesting. Thank you for this additional information, it was needed to properly understand your problem. I was able to add the ISBN by manually editing the quickadd user script mentioned in the link you shared. See the edit in the image, it's just one additional line. 

<img width="377" alt="quickadd_script_edit" src="https://user-images.githubusercontent.com/32915653/193837866-38665d93-3dcb-468f-b6b3-f3ebbd0cc4d6.png">

With that added in the user script, I also added `{{VALUE:isbn}}` in the template that quickadd uses. With those two changes, the ISBN (if it exists) autofills like the other properties when quickadd is triggered. Hopefully these two changes work for you as well.
